docs/01-app/01-getting-started/15-backend-for-frontend.mdx

```diff
@@ -0,0 +1,787 @@
+---
+title: How to use Next.js as a backend for your frontend
+nav_title: Backend
+description: Learn how to use Next.js as a backend framework
+related:
+  title: API Reference
+  description: Learn more about Route Handlers and Middleware
+  links:
+    - app/building-your-application/routing/route-handlers
+    - app/api-reference/file-conventions/route
+    - app/building-your-application/routing/middleware
+---
+
+With Next.js you can achieve a pattern known as "Backend for Frontend". This pattern makes it possible, to use the Next.js server to build publicly reachable endpoints, to receive any type of HTTP requests, and deliver any type of content, not just HTML. It is also possible to access other data sources, and update remote data, amon other side effects, within this backend for your frontend.
```

Comment:
We have a `--api` template with create-next-app, it might be good to mention it here at the start.